self-paced-labs/vertex-ai/vertex-ai-qwikstart/lab_exercise.ipynb

```diff
@@ -1734,127 +2110,188 @@
     "Note: this `CustomContainerTrainingJob` will take about 20 minutes to provision resources and train your model."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 1432,
+   "id": "1690db87",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# command line args for trainer.task defined above. Review the 'help' argument for a description.\n",
+    "# You will set the model training args below. Vertex AI will set the environment variables for training URIs.\n",
+    "CMD_ARGS= [\n",
+    "    \"--learning-rate=\" + str(0.001),\n",
+    "    \"--batch-size=\" + str(16),\n",
+    "    \"--n-train-examples=\" + str(2638),\n",
+    "    \"--stop-point=\" + str(10),\n",
+    "    \"--n-checkpoints=\" + str(10),\n",
+    "    \"--dropout=\" + str(0.2),   \n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1433,
+   "id": "b490e0e8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "TIMESTAMP=datetime.datetime.now().strftime('%Y%m%d%H%M%S')\n",
+    "BASE_OUTPUT_DIR= f\"gs://{GCS_BUCKET}/vertex-custom-training-{MODEL_NAME}-{TIMESTAMP}\""
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "1fea2be9",
+   "id": "c6380811",
    "metadata": {},
    "outputs": [],
    "source": [
     "job = aiplatform.CustomContainerTrainingJob(\n",
-    "    display_name=\"online-retail-clv\",\n",
-    "    container_uri=f\"gcr.io/{PROJECT_ID}/online-retail-clv:latest\",\n",
+    "    display_name=\"online-retail-clv-3M-dnn-regressor\",\n",
+    "    container_uri=IMAGE_URI,\n",
     "    # https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers\n",
```

Comment:
Either we should use true pre-built containers, from us-docker.pkg.dev/vertex-ai/training and then use CustomTrainingJob or CustomPythonPackageTrainingJob, or pre-built containers are not relevant here